src/main/java/dev/snowdrop/release/App.java

```diff
@@ -350,8 +357,11 @@ public void status(
     }
 
     private BasicIssue clone(Release release, String token) throws IOException {
-        final var issue = service.clone(release, IssueService.RELEASE_TICKET_TEMPLATE, watchers);
-        release.setJiraKey(issue.getKey());
+        BasicIssue issue = null;
+        if (!release.isTestMode()) {
+            issue = service.clone(release, IssueService.RELEASE_TICKET_TEMPLATE, watchers);
+            release.setJiraKey(issue.getKey());
+        }
```

Comment:
So, I would prefer to return an Optional<BasicIssue> here and avoid to set BasicIssue to null.

Reply:
I left the clone method as was, without the check for test mode, because on test mode the clone is also executed but using a dummy JIRA project. This is defined prior to this call in the `release.setTest(test);` statement of the setup release command.

see https://github.com/snowdrop/release-manager/blob/04b8e30e4a305485fe36555c310093cf048875a5/src/main/java/dev/snowdrop/release/App.java#L221